ci/pipeline.yml

```diff
@@ -273,7 +271,7 @@ jobs:
         params:
           ATC_EXTERNAL_URL: ((concourse-url))
           FLY_USERNAME: ((concourse-username))
-          FLY_PASSWORD:  ((concourse-password))
+          FLY_PASSWORD:  '((concourse-password))'
```

Comment:
Maybe we can create an issue to adds quotes to password into our pipeline templates